sentinel-adapter/sentinel-spring-webflux-adapter/src/test/java/com/alibaba/csp/sentinel/adapter/spring/webflux/SentinelWebFluxIntegrationTest.java

```diff
@@ -71,6 +71,30 @@ public void testWebFluxFilterBasic() throws Exception {
         assertEquals(1, cn.passQps(), 0.01);
     }
 
+    @Test
+    public void testWebFluxRouterFunction() throws Exception {
+
+        String url = "/router/hello";
```

Comment:
I don't think this url is configured by  the routerFunction way.
yes, the test case works fine, but it can not prove the adaptor works well with routerFunction.

Reply:
PRs are welcomed if there are any unresolved problems.

Reply:
Do you mean run a server like this?@lawulu
```java
HttpHandler httpHandler = RouterFunctions.toHttpHandler(route);
HttpServlet servlet = new ServletHttpHandlerAdapter(httpHandler);
Tomcat server = new Tomcat();
Context rootContext = server.addContext("",
  System.getProperty("java.io.tmpdir"));
Tomcat.addServlet(rootContext, "servlet", servlet);  
rootContext.addServletMapping("/", "servlet");
tomcatServer.start();
```

Reply:
No，you can refer to https://docs.spring.io/spring-framework/docs/5.0.0.BUILD-SNAPSHOT/spring-framework-reference/html/web-reactive.html
There is an example: 
`RouterFunction<ServerResponse> helloWorldRoute =
	RouterFunctions.route(RequestPredicates.path("/hello-world"),
	request -> Response.ok().body(fromObject("Hello World")));
`
In this way, the  filter based on `WebFilter` does not work. Users should use `HandlerFilterFunction` instead.
Unfortunately, there is no ready solution。

Reply:
Could you please provide a demo for this?